payments-core/src/main/java/com/stripe/android/Stripe.kt

```diff
@@ -1847,8 +1848,17 @@ class Stripe internal constructor(
          *
          * See [Building Stripe Plug-ins and Libraries - Setting the API version](https://stripe.com/docs/building-plugins#setappinfo).
          */
+        @Deprecated(
+            message = "Use StripeAppInfo.value instead",
+            replaceWith = ReplaceWith(
+                expression = "StripeAppInfo.value",
+                imports = ["com.stripe.android.core.StripeAppInfo"]
+            )
+        )
         @JvmStatic
-        var appInfo: AppInfo? = null
+        var appInfo: AppInfo?
+            set(value) { StripeAppInfo.value = value }
+            get() = StripeAppInfo.value
```

Comment:
non-payments SDKs should also expose a global `appInfo` field, but `Stripe` lives in payments. Added `StripeAppInfo` to `stripe-core` replicating `Stripe.appInfo`.